Libplanet/Net/NetMQTransport.cs

```diff
@@ -922,6 +907,44 @@ private async Task CreatePermission(BoundPeer peer)
                     throw;
                 }
             }
+            catch (Exception e) when (e is SocketException || e is ObjectDisposedException)
+            {
+                _logger.Error($"Unexpected error occurred during {nameof(CreatePermission)}: {e}");
+
+                if (!(_turnClient is null))
+                {
+                    _logger.Debug("Trying to re-connect to TURN...");
+
+                    _turnClient.Dispose();
+                    _turnCancellationTokenSource.Cancel();
+                    await Task.WhenAny(_turnTasks);
+                    await ConnectToTurnClient();
+                }
+            }
+        }
+
+        private async Task ConnectToTurnClient()
```

Comment:
`ConntectToTurnClient` seems a little inappropriate because it doesn't just connect to the turn client, it's also responsible for initializing it. how about `InitalizeTurnClient` ?